delete.go

```diff
@@ -39,7 +42,41 @@ EXAMPLE:
 		},
 	},
 	Action: func(context *cli.Context) error {
-		// TODO
-		return nil
+		return delete(context.String("container-id"), context.Bool("force"))
 	},
 }
+
+func delete(containerID string, force bool) error {
+	// Checks the MUST and MUST NOT from OCI runtime specification
+	if err := checkDeleteParamsIntegrity(containerID); err != nil {
+		return err
+	}
+
+	if _, err := vc.StopPod(containerID); err != nil {
+		return err
+	}
+
```

Comment:
@sboeuf Yes, that's the runc approach as well. We need to actually update our container status file as well, through a virtcontainers `UpdateStatus()` container API. It should be called for our `List` APIs too.
Please do that as a follow up PR between runtime and virtcontainers.